src/plugin/maven/src/main/kotlin/BaseMojo.kt

```diff
@@ -34,13 +30,9 @@ abstract class BaseMojo : AbstractMojo() {
      * Multiple paths can be provided, separated by commas.
      * Files can also be loaded from the classpath using the 'classpath:' prefix (e.g., 'classpath:wirespec/petstore.ws').
      */
-
     @Parameter(required = true)
     protected lateinit var input: String
 
-    /**
```

Comment:
Why are the comments removed. These comments are visible while using the maven plugin.

Reply:
Check, added back in and also for other properties